lib/log-formatters.js

```diff
@@ -80,19 +79,20 @@ module.exports = function (d) {
    * Format an X-Trace ID in easier to look at format.
    */
   function humanID (x) {
-    if (x instanceof ao.addon.Event) {
+    if (x.constructor.name === 'Event' && typeof x.event === 'undefined') {
```

Comment:
Does `x.toString(1)` also work like `x.event.toString(1)` where the parameter 1 is afaict a type of formatting flag?

Reply:
Yes both do. `1` is indeed a formatting flag. [It goes down to the bindings](https://github.com/appoptics/appoptics-bindings-node/blob/master/src/event/event-to-string.cc). There are more formatting options. Test covering them was added [recently](https://github.com/appoptics/appoptics-bindings-node/pull/94). The whole thing is overcomplicated. However, once such an implementation is done and gets "baked in". Removing it becomes a complex refactoring task by itself.